tools/amd_build/pyHIPIFY/cuda_to_hip_mappings.py

```diff
@@ -0,0 +1,2097 @@
+     "CU_MEMORYTYPE_HOST": ("hipMemTypeHost", CONV_TYPE, API_DRIVER,HIP_UNSUPPORTED),
+     "CU_MEMORYTYPE_DEVICE": ("hipMemTypeDevice", CONV_TYPE, API_DRIVER,HIP_UNSUPPORTED),
+     "CU_MEMORYTYPE_ARRAY": ("hipMemTypeArray", CONV_TYPE, API_DRIVER,HIP_UNSUPPORTED),
+     "CU_MEMORYTYPE_UNIFIED": ("hipMemTypeUnified", CONV_TYPE, API_DRIVER,HIP_UNSUPPORTED),
+     "CU_RESOURCE_TYPE_ARRAY": ("hipResourceTypeArray", CONV_TEX, API_DRIVER,HIP_UNSUPPORTED),
+     "CU_RESOURCE_TYPE_MIPMAPPED_ARRAY": ("hipResourceTypeMipmappedArray", CONV_TEX, API_DRIVER,HIP_UNSUPPORTED),
+     "CU_RESOURCE_TYPE_LINEAR": ("hipResourceTypeLinear", CONV_TEX, API_DRIVER,HIP_UNSUPPORTED),
+     "CU_RESOURCE_TYPE_PITCH2D": ("hipResourceTypePitch2D", CONV_TEX, API_DRIVER,HIP_UNSUPPORTED),
+     "CU_RES_VIEW_FORMAT_NONE": ("hipResViewFormatNone", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_UINT_1X8": ("hipResViewFormatUnsignedChar1", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_UINT_2X8": ("hipResViewFormatUnsignedChar2", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_UINT_4X8": ("hipResViewFormatUnsignedChar4", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_SINT_1X8": ("hipResViewFormatSignedChar1", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_SINT_2X8": ("hipResViewFormatSignedChar2", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_SINT_4X8": ("hipResViewFormatSignedChar4", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_UINT_1X16": ("hipResViewFormatUnsignedShort1", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_UINT_2X16": ("hipResViewFormatUnsignedShort2", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_UINT_4X16": ("hipResViewFormatUnsignedShort4", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_SINT_1X16": ("hipResViewFormatSignedShort1", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_SINT_2X16": ("hipResViewFormatSignedShort2", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_SINT_4X16": ("hipResViewFormatSignedShort4", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_UINT_1X32": ("hipResViewFormatUnsignedInt1", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_UINT_2X32": ("hipResViewFormatUnsignedInt2", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_UINT_4X32": ("hipResViewFormatUnsignedInt4", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_SINT_1X32": ("hipResViewFormatSignedInt1", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_SINT_2X32": ("hipResViewFormatSignedInt2", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_SINT_4X32": ("hipResViewFormatSignedInt4", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_FLOAT_1X16": ("hipResViewFormatHalf1", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_FLOAT_2X16": ("hipResViewFormatHalf2", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_FLOAT_4X16": ("hipResViewFormatHalf4", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_FLOAT_1X32": ("hipResViewFormatFloat1", CONV_TEX,API_DRIVER),
+     "CU_RES_VIEW_FORMAT_FLOAT_2X32": ("hipResVi
```

Comment:
Can we sort these before the initial commit? Will make it easier to find things / make modifications later.